frontend/app-development/features/processEditor/handlers/OnProcessTaskAddHandler.ts

```diff
@@ -32,4 +77,38 @@ export class OnProcessTaskAddHandler {
       rules: [...this.currentPolicy.rules, ...defaultPaymentPolicy.rules],
     });
   }
+
+  /**
+   * Adds a dataType to the added signing task
+   * @param taskMetadata
+   * @private
+   */
+  private handleSigningTaskAdd(taskMetadata: OnProcessTaskEvent): void {
+    // Add DataType
```

Comment:
Is this needed? 😄

Reply:
Since this method performs multiple tasks, I added comments to each "thing" to increase readability by creating a "visual section" and clarify what happens within the method.

Reply:
I still think the name of the function is self-explaining as it says `addDataType...` 😅 
Also, the JS doc on the function says `Adds a datatype`, and as this function does not do anything more that adding, I believe that is enough. What do you think?